sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -279,7 +282,7 @@ abstract class BinaryArithmetic extends BinaryOperator
   }
 
   /** Name of the function for this expression on a [[Decimal]] type. */
-  def decimalMethod: String =
+  def decimalMethod(mathContextValue: GlobalValue, value1: String, value2: String): String =
```

Comment:
Where are the parameters `mathContextValue`, `value1`, `value2` used for?

Reply:
I guess this change no need.